Userland/Services/WebSocket/ConnectionFromClient.cpp

```diff
@@ -75,6 +75,15 @@ Messages::WebSocketServer::ReadyStateResponse ConnectionFromClient::ready_state(
     return (u32)ReadyState::Closed;
 }
 
+Messages::WebSocketServer::SubprotocolInUseResponse ConnectionFromClient::subprotocol_in_use(i32 connection_id)
+{
+    RefPtr<WebSocket> connection = m_connections.get(connection_id).value_or({});
+    if (connection) {
+        return (DeprecatedString)connection->subprotocol_in_use();
+    }
+    return (DeprecatedString)DeprecatedString::empty();
```

Comment:
Again these casts look funky, also `DeprecatedString::empty()` for sure won't need a cast to `DeprecatedString` :^)